src/modules/material-requests/components/MaterialRequestDetailBlade.tsx

```diff
@@ -0,0 +1,152 @@
+import { Button } from '@meemoo/react-components';
+import Image from 'next/image';
+import { FC } from 'react';
+
+import { Blade, Icon, IconNamesLight } from '@shared/components';
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+import { formatMediumDate } from '@shared/utils';
+
+import { MaterialRequestDetails } from '@material-requests/types';
+
+interface MaterialRequestDetailBladeProps {
```

Comment:
Consistency: Put this file inside a folder called `MaterialRequestDetailBlade` that has its own types, css module and const file if needed. You can use `modules > account > components` as comparison.